src/quest/persistence.py

```diff
@@ -21,9 +22,10 @@ def delete_blob(self, key: str): ...
 
 
 class PersistentHistory(History):
-    def __init__(self, namespace: str, storage: BlobStorage):
+    def __init__(self, namespace: str, storage: BlobStorage, masterserializer: MasterSerializer = None):
         self._namespace = namespace
         self._storage = storage
+        self._masterserializer = masterserializer if masterserializer is not None else MasterSerializer()
```

Comment:
```py
self._master_serializer = master_serializer or MasterSerializer()
```

Reply:
Actually, if the dev isn't passing in a customized MasterSerializer, we should have a dummy default that simply returns the object as-is (to avoid the unnecessary overhead of traversing the data).